x-pack/functionbeat/config/config.go

```diff
@@ -134,3 +138,40 @@ func (p *ProviderConfig) Validate() error {
 	}
 	return nil
 }
+
+// MemSizeFactor64 implements a human understandable format for bytes but also make sure that all
+// values used are a factory of 64.
+type MemSizeFactor64 int
+
+// Unpack converts a size defined from a human readable format into bytes and ensure that the value
+// is a factoru of 64.
```

Comment:
s/factoru/multiple/

This functions does not ensure that we have a multiple of 64, which would require us to round up by computing `((sz + 63) / 64) * 64`, but it verifies and errors if we don't have a multiple of 64.